nova/tests/unit/virt/vmwareapi/test_driver_api.py

```diff
@@ -242,9 +242,17 @@ def setUp(self, mock_svc, mock_register):
         """
         self.useFixture(
             fixtures.MonkeyPatch(
-                'nova.utils.vm_needs_special_spawning',
+                'nova.utils.spawn',
+                self._fake_spawn))
+
+        self.useFixture(
+            fixtures.MonkeyPatch(
+                'nova.utils.',
```

Comment:
I think this is missing the `vm_needs_special_spawning` now. Did it work without that? Do we still need it?

Reply:
```
        self.useFixture(
            fixtures.MonkeyPatch(
                'nova.utils.spawn',
                self._fake_spawn))

        self.useFixture(
            fixtures.MonkeyPatch(
                'nova.utils.',
                self._fake_vm_needs_special_spawning))
```
This is the new code. One is patching `nova.utils.spawn` with `self._fake_spawn`. The second is patching `nova.utils.` with `self._fake_vm_needs_special_spawning`. My point is, that `nova.utils.` should be `nova.utils.vm_needs_special_spawning`.

Reply:
Ah, sorry. You're right. I've fixed it.